samples/bluetooth/peripheral_lbs/README.rst

```diff
@@ -25,6 +26,19 @@ Requirements
 User interface
 **************
 
+For nRF52840 Dongle:
+
+RGB LED:
+   Red:
+      * On when connected
+   Green:
+      * Controlled remotely from the connected device.
```

Comment:
It's a bit unclear to me, so if remote device turn off led by writting to the characteristic then the LED is red or does not shine ? Maybe It is worth to explain it more precisely.

Reply:
Just a red part of the RGB LED would be active in this case.
The user simply controls the Green part of the RGB LED with the LBS characteristic.
RGB LED parts are activated independently (Red state doesn't affect Green state) like standard LEDs on the DK (LED3 state doesn't affect the LED2 state).

In the UI description, I am referring to the RGB LED part that is triggered by a given event (connected, characteristics write). I am not describing the colour of the RGB LED on the output, this could be a combination of Red and Green colour.

I can provide additional explanation if you still think this is required. Could you provide a sentence that would make it more clear to you?

Reply:
I added an additional explanation in the testing steps where it is described what happens when the user writes to the LBS characteristic.

Reply:
Ok thanks.